src/main/java/org/mockito/internal/session/DefaultMockitoSessionBuilder.java

```diff
@@ -0,0 +1,33 @@
+package org.mockito.internal.session;
+
+import org.mockito.MockitoSession;
+import org.mockito.internal.framework.DefaultMockitoSession;
+import org.mockito.internal.util.ConsoleMockitoLogger;
+import org.mockito.quality.Strictness;
+import org.mockito.session.MockitoSessionBuilder;
+
+public class DefaultMockitoSessionBuilder implements MockitoSessionBuilder {
+
+    private Object testClassInstance;
+    private Strictness strictness;
+
+    @Override
+    public MockitoSessionBuilder initMocks(Object testClassInstance) {
```

Comment:
About ROI : `mockitoSession` will probably be more especially useful to framework developers. `initMocks` is for them the way to initialize mockito annotations. Or other annotations if they plug another `AnnotationEngine`. If we were to introduce other annotations that are not about mocks, it will feel awkward.

If we ever implement a way to inject non mock in the `@InjectMocks` field then `initMocks` will be even more awkward. If we add other annotations that non mock related, but for Mockito lifecycle it'll be even more _off_.